tests/framework/checks/checks.go

```diff
@@ -129,3 +130,28 @@ func IsRunningOnKindInfra() bool {
 	provider := os.Getenv("KUBEVIRT_PROVIDER")
 	return strings.HasPrefix(provider, "kind")
 }
+
+func IsThereEnoughNodesWithCPUManager(nodeCount int) {
```

Comment:
thanks for removing code from tests/framework/checks/skips.go. We should not have programmatic skips, they end up with us not running tests and not being aware of it.

However, I would expect a function named `Is*` to return a `bool`. How about `RequireCPUManagerNodes`?